hack/manifests/csi/vsphere/05-example-pod.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: example-windows-pod
+  name: example-windows-pod
+  # must be in the same namespace as the PVC
+  namespace: windows-storage-example
+spec:
+  selector:
+    matchLabels:
+      app: example-windows-pod
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: example-windows-pod
+      name: example-windows-pod
+    spec:
```

Comment:
@alinaryan, The `os` field got skipped in [this](https://github.com/openshift/windows-machine-config-operator/compare/bda14fef971b55839baf9f8553ca5b2837c4cb0e..1cb901ef1ee536994b7f65edc9740a65a2ced079#diff-e11646292771aad6465ca3da31522c869be8733b1099542ef50ca43bb4b8cec3L8-L9) push.  I would suggest that you bring it back.